jobs-shell/script.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloudrun_jobs_shell_script]
+set -euox pipefail
```

Comment:
Let's remove the "x", the extra print outs may be distracting

Reply:
I agree with distracting, but I didn't want to remove it because if you run script without -x, they become harder to debug if they fail.

Reply:
Added a note in comments

Reply:
thanks